index.html

```diff
@@ -25,15 +25,39 @@
             <i class="close-item fa fa-times"></i>
             <div class="mob-nav-div">
                 <ul class="mobile-nav">
-                    <li class="aboutbtn"><a href="#recnt-projects">About</a></li>
+                    <li class="aboutbtn"><a href="about.html">About</a></li>
                     <li class="programsbtn"><a href="#programs" >Programs</a></li>
                     <li class="speakersbtn"><a href="#speakers" >Speakers</a></li>
                     <li class="sponsorsbtn"><a href="#sponsors" >Sponsors</a></li>
                     <li class="contactbtn"><a href="#contact" >Contact us</a></li>
-                    <li class="conferencebtn"><a href="#conference" >Conference</a></li>
+                    <li class="conferencebtn"><a href="/" >Conference</a></li>
                 </ul>
             </div>
         </div>
+        <div class="first-header">
+            <ul class="first-header-list">
+              <li><a href="/" aria-label="facebook"><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href="/" aria-label="Twitter"><i class="fab fa-twitter"></i></a></li>
+              <li><a href="/">English</a></li>
+              <li><a href="/">My Page</a></li>
+              <li><a href="/">Logout</a></li>
+            </ul> 
+        </div>
```

Comment:
- [ ] Please, kindly fix the white space on the top of the page, before the section with social media icons. By fixing it your project will be closer to the original design and will look much more professional.

Please, also consider spaces on the left and right sides of the mentioned section. You can see the width of the `div` does not cover all page widths.

> You version:
<img width="1437" alt="Screen Shot 2022-02-03 at 01 59 57" src="https://user-images.githubusercontent.com/49064106/152244847-047c1ad1-4940-4447-882e-47844f7de8b7.png">

> Original design:
<img width="985" alt="Screen Shot 2022-02-03 at 01 59 45" src="https://user-images.githubusercontent.com/49064106/152244913-fa6b5be9-0778-45d9-b4b0-b2027e998f65.png">